### PR TITLE
FSDK-85: TypeScript Gallery

### PR DESCRIPTION
**Pull request by arnautov-anton**

### CODE REVIEW

File: src/components/Gallery.mdx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { Playground, Props } from 'docz';
+import { Gallery } from './Gallery';
+
+# Gallery
+
+## Properties
+
+<Props of={Gallery} />
+
+## Basic usage
```
Comment (mahboubii) [inline src/components/Gallery.mdx:15]: missing single, and double images case

File: src/components/Gallery.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import React, { useState, useMemo } from 'react';
+import Lightbox from 'react-images';
```
Comment (mahboubii) [inline src/components/Gallery.tsx:2]: good moment to upgrade this dependency as well

File: src/components/Gallery.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  return (
+    <div className="raf-gallery">
+      {images.slice(0, 5).map((image, i) => (
+        <div
+          role="button"
+          className={`img ${i === 4 && images.length > 5 ? 'img--last' : ''}`}
+          onClick={() => setCurrentImage(i)}
+          key={`image-${i}`}
+        >
+          <img src={image} className={`raf-gallery__image `} alt="" />
```
Comment (mahboubii) [inline src/components/Gallery.tsx:24]:
```suggestion
          <img src={image} className="raf-gallery__image" alt="" />
```

File: src/components/Gallery.tsx [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+          <React.Fragment>{i === 4 && images.length > 5 ? <p>{images.length - 4} more</p> : null}</React.Fragment>
```
Comment (mahboubii) [inline src/components/Gallery.tsx:25]:
```suggestion
          <React.Fragment>{i === 4 && images.length > 5 && <p>{images.length - 4} more</p>}</React.Fragment>
```

File: src/components/Gallery.tsx (hunk 2 above)
Comment (arnautov-anton) [reply src/components/Gallery.tsx:2]: Looks like it's no longer maintained and the last available [version](https://www.npmjs.com/package/react-images) is in beta.

Comment (mahboubii) [reply src/components/Gallery.tsx:2]: our use case is super simple, let's first see if upgrading it is gonna worth it or not, alternatively we can replace it if there is a similar/lighter lib

**Review by mahboubii — APPROVED** [review]

**Pull request merged**